src/main/java/uk/ac/ebi/eva/pipeline/configuration/validation/VepAnnotationGeneratorStepParametersValidator.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2016 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.pipeline.configuration.validation;
+
+import org.springframework.batch.core.JobParameters;
+import org.springframework.batch.core.JobParametersInvalidException;
+import org.springframework.batch.core.job.DefaultJobParametersValidator;
+
+import uk.ac.ebi.eva.pipeline.configuration.JobParametersNames;
+
+/**
+ * Validates the job parameters necessary to execute a {@link uk.ac.ebi.eva.pipeline.jobs.steps.VepAnnotationGeneratorStep}
+ * <p>
+ * The parameters OUTPUT_DIR_ANNOTATION, INPUT_STUDY_ID and INPUT_VCF_ID are used to build the vep input/output options
+ */
+public class VepAnnotationGeneratorStepParametersValidator extends DefaultJobParametersValidator {
+
+    private StepParametersValidatorUtil parametersValidator;
```

Comment:
Could it be an improvement to create the following class hierarchy instead of delegating?

`DefaultJobParametersValidator <- StepParametersValidatorUtil <- VepAnnotationGeneratorStepParametersValidator`

In general inheritance loses vs composition, but in this case it would heavily reduce the verbosity inside the `validate` methods. On the other hand, `StepParametersValidatorUtil` would turn into an abstract class and we would need a mock to test it.

I think it's worth discussing @poggio84 @pabarcgar @jorizci